config/routes.rb

```diff
@@ -17,6 +17,7 @@
       member do
         get 'refresh'
         post 'refresh'
+        get 'courses.json', to: 'courses#show_json', format: 'json', as: 'one_course_json'
```

Comment:
is the `.json` required? (we have also the format)

so this would map to
`/courses/:id/course.json`? why not just `/courses/:id.json`